dev_guide/application_lifecycle.adoc

```diff
@@ -64,14 +74,20 @@ Generate::
 and deployment configurations.
 
 Verify::
-* Verify and test the application code is running successfully once the build
-completes.
+* Depending on your preferences, you have a number of ways to verify your
+application. You can verify by pushing your changes to your application's Github
+repository, and use OpenShift to rebuild and redeploy your application.
+Alternatively, you can use hot deploy with rsync to synchronize your code
+changes into a running pod.
+* You can test locally, then push your changes to GitHub, then build and deploy
+the entire application to verify. For example, if you are using a Java
+application and Eclipse.
```

Comment:
again this is the "develop locally" flow, so none of this would apply.  Verification would consist of making sure the application that was built+deployed by the "generate" step is running correctly in openshift.